src/components/app/Overview/Overview.tsx

```diff
@@ -439,6 +439,7 @@ export default function AppOverview({ appMetaInfo, getAppMetaInfoRes, isJobOverv
                                 className="mr-16 w-150 h-20 m-tb-8 fs-13 cn-9 flex dc__content-start"
                             >
                                 {environmentName(jobPipeline)}
+                                {environmentName(jobPipeline) === "devtron-ci" && <span className="fw-4 fs-11 ml-4 dc__italic-font-style" >{`(Default)`}</span>}
```

Comment:
we should make a constant of `devtron-ci` and use it everywhere

Reply:
done